Source/WebKit/WPEPlatform/wpe/WPEExtensions.cpp

```diff
@@ -57,7 +58,16 @@ void wpeEnsureExtensionPointsLoaded()
 
     static std::once_flag onceFlag;
     std::call_once(onceFlag, [] {
-        // FIXME: load external modules.
+        GIOModuleScope* scope = g_io_module_scope_new(G_IO_MODULE_SCOPE_BLOCK_DUPLICATES);
+        const char* path = g_getenv("WPE_PLATFORMS_PATH");
+        if (path && *path) {
+            GUniquePtr<char*> paths(g_strsplit(path, G_SEARCHPATH_SEPARATOR_S, 0));
+            for (int i = 0; paths.get()[i]; ++i)
```

Comment:
Array indexes cannot be negative, use an unsigned type:

```suggestion
            for (size_t i = 0; paths.get()[i]; ++i)
```

Also, we may want to check each path here with `g_file_query_file_type() == G_FILE_TYPE_DIRECTORY` before scanning it.

Reply:
hmm, first thing `g_io_modules_scan_all_in_directory_with_scope` does is `g_dir_open` and returns early when it fails.

Reply:
Nice, then it's less work to do o our end 👏🏼